ansible/Dockerfile

```diff
@@ -1,5 +1,23 @@
 FROM alpine:3.6
-RUN apk update
-RUN apk add git \
-    ansible
-RUN echo "'Done !"
\ No newline at end of file
+
+ARG CI_HELPER_VERSION
+
+WORKDIR /root
+RUN apk update &&\
+ apk add -q --no-cache 
+RUN apk add bash 
+RUN apk add ca-certificates 
+RUN apk add curl 
+RUN apk add git 
+RUN apk add openssh-client 
+RUN apk add python 
+RUN apk add py-pip 
+RUN apk add ansible
```

Comment:
In travis.yml you specified ansible VERSION=2.3.2, but you don't lock your version here. 
The test will break whenever ansible releases a new version.
You should use $VERSION to explicitly install that specific release, and avoid regressions.